src/index.html

```diff
@@ -1,42 +1,53 @@
 <!DOCTYPE html>
 <html>
-
 <head>
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
     <title>Basic Web Video Conference Application</title>
-    <script src="https://unpkg.com/@voxeet/voxeet-web-sdk@2.0.0" type="text/javascript"></script>
-    <script src="./scripts/ui.js" type="text/javascript"></script>
+    <!-- Get the latest version of the SDK at
+         https://dolby.io/developers/interactivity-apis/client-sdk/release-notes/changelog -->
+    <script type="text/javascript" src="https://unpkg.com/@voxeet/voxeet-web-sdk@2.4.3"></script>
```

Comment:
can we remove the version to avoid repeatedly updating this sample and always use the latest version?

Reply:
I completely understand, unpkg has a way to request the latest version, I'm going to use that instead.